tools/pydocgen/pulumi-docs.json

```
@@ -115,6 +115,12 @@
       "pulumi_provider_name": "keycloak",
       "terraform_provider_name": ""
     },
+    {
+      "name": "Pulumi Kong",
+      "package_name": "pulumi_kong",
+      "pulumi_provider_name": "kong",
+      "terraform_provider_name": "kong"
```

We leave this blank when the TF provider isn't in the https://github.com/terraform-providers GH org.

```suggestion
      "terraform_provider_name": ""
```

---

Ah ok. I'll update it.